recipes/hlama/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: hlama
+  version: '0.3'
+
+build:
+  number: 0
+  skip: True # [py27 or py35 or osx]
```

Comment:
Try to skip py3k. I will look at it in the evening.
On Sep 14, 2016 8:54 AM, Manuel Holtgrewe <notifications@github.com> wrote: